src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -35,12 +39,52 @@ public Settings(IConfigurationSource source)
             JaegerExporterAgentHost = source.GetString(ConfigurationKeys.JaegerExporterAgentHost) ?? "localhost";
             JaegerExporterAgentPort = source.GetInt32(ConfigurationKeys.JaegerExporterAgentPort) ?? 6831;
 
+            LoadTracerAtStartup = source.GetBool(ConfigurationKeys.LoadTracerAtStartup) ?? true;
+
+            ConsoleExporterEnabled = source.GetBool(ConfigurationKeys.ConsoleExporterEnabled) ?? true;
+
+            EnabledIntegrations = Enum.GetValues(typeof(IntegrationIds)).Cast<IntegrationIds>().ToList();
+            var disabledIntegrations = source.GetString(ConfigurationKeys.DisabledIntegrations);
+            if (disabledIntegrations != null)
+            {
+                foreach (var instrumentation in disabledIntegrations.Split(separator: ','))
+                {
+                    if (Enum.TryParse(instrumentation, out IntegrationIds parsedType))
+                    {
+                        EnabledIntegrations.Remove(parsedType);
+                    }
+                    else
+                    {
+                        // TODO replace with proper logging
+                        _logger.Warning($"Could not parse instrumentation \"{instrumentation}\". Skipping...");
+                    }
+                }
+            }
+
+            var additionalSources = source.GetString(ConfigurationKeys.AdditionalSources);
+            if (additionalSources != null)
```

Comment:
Don't change this for this PR but this code is repeated for these "enumerative" configurations, it seems to deserve common code to handle it. Anyway, not on this PR.